### PR TITLE
remove ASCEND* keyword

### PR DESCRIPTION
**Pull request by GreatV**
### PR types
Others

### PR changes
Others

### Describe

remove ASCEND* keyword

### CODE REVIEW

File: tools/gen_ut_cmakelists.py [hunk 1]
```diff
@@ -100,8 +100,6 @@ def _proccess_archs(arch):
             assert a in [
                 "GPU",
                 "ROCM",
-                "ASCEND",
-                "ASCEND_CL",
                 "XPU",
             ], f"""Supported arhc options are "GPU", "ROCM", "ASCEND" and "ASCEND_CL", "XPU", but the options is {a}"""
```
Comment (luotao1) [inline tools/gen_ut_cmakelists.py]: `Supported arhc options are "GPU", "ROCM", "ASCEND" and "ASCEND_CL", "XPU", but the options is`里的关键字也可以清一下

**Review by ronny1996 — APPROVED** [review]
LGTM

**Comment by luotao1** [comment]
还有以下一些ASCEND的关键字，看能否清理： @GreatV 
https://github.com/PaddlePaddle/Paddle/blob/7b5065abffd569e34f8131790f3e85bed30d726a/python/paddle/fluid/tests/unittests/CMakeLists.txt#L41-L42
- python/paddle/fluid/tests/unittests/collective/README.md
- python/paddle/fluid/tests/unittests/collective/fleet/testslist.csv
- python/paddle/fluid/tests/unittests/collective/testslist.csv

**Comment by GreatV** [comment]
> 还有以下一些ASCEND的关键字，看能否清理： @GreatV
> 
> https://github.com/PaddlePaddle/Paddle/blob/7b5065abffd569e34f8131790f3e85bed30d726a/python/paddle/fluid/tests/unittests/CMakeLists.txt#L41-L42
> 
> * python/paddle/fluid/tests/unittests/collective/README.md
> * python/paddle/fluid/tests/unittests/collective/fleet/testslist.csv
> * python/paddle/fluid/tests/unittests/collective/testslist.csv

好的，下个PR清理。

**Pull request merged**